fix linking issues that manifest themselves as "ld: cannot find -lpcm, ld: cannot find -lgetkw"
author: Kenneth Hoste (HPC-UGent)
--- DIRAC-14.1-Source/cmake/ConfigExternal.cmake.orig	2015-12-11 16:36:08.080428412 +0100
+++ DIRAC-14.1-Source/cmake/ConfigExternal.cmake	2015-12-11 16:36:54.901482233 +0100
@@ -43,6 +43,8 @@
 
     link_directories(${PROJECT_BINARY_DIR}/external/lib)
     link_directories(${PROJECT_BINARY_DIR}/external/${_project}-build/external/lib)
+    link_directories(${PROJECT_BINARY_DIR}/external/${_project}-build/)
+    link_directories(${PROJECT_BINARY_DIR}/external/${_project}-build/external/libgetkw-build/C++)
 
     add_dependencies(${_project} check_external_timestamp_${_project})
 
